app/src/main/java/de/xikolo/managers/DownloadManager.kt

```diff
@@ -173,14 +188,47 @@ class DownloadManager(activity: FragmentActivity) {
         }
     }
 
-    fun getDownload(downloadAsset: DownloadAsset): Download? =
-        DownloadService.getInstance()?.getDownload(downloadAsset.url)
+    fun getDownloadTotalBytes(downloadAsset: DownloadAsset): Long {
+        var totalBytes = 0L
+        val mainDownload = DownloadService.getInstance()?.getDownload(downloadAsset.url)
+        totalBytes +=
+            if (mainDownload != null && mainDownload.totalBytes > 0L)
```

Comment:
```suggestion
            if (mainDownload?.totalBytes > 0L)
```
Does this work?

Reply:
```suggestion
            if (mainDownload != null && mainDownload.totalBytes > 0L)
```
Works only like `mainDownload?.totalBytes?.compareTo(0L)!! > 0`